distributed/worker.py

```diff
@@ -2288,6 +2291,15 @@ def select_keys_for_gather(self, worker, dep):
 
         return deps, total_bytes
 
+    @property
+    def total_comm_bytes(self):
+        warnings.warn(
+            "The attribute `Worker.total_comm_bytes` has been renamed to `comm_threshold_bytes`. "
+            "Future versions will only support the new name.",
+            DeprecationWarning,
+        )
```

Comment:
1. The `dask.utils._deprecated` decorator does not work on properties since properties don't have a __name__ attribute
2. I used `DeprecationWarning` instead of `FutureWarning` since they are recommended for devs instead of users. See also https://stackoverflow.com/a/55378483